climpact2.batch.stations.r

```diff
@@ -96,15 +95,14 @@ batch <- function(input.directory,output.directory,base.start,base.end) {
 					fileConn<-file(paste(file,".error.txt",sep=""))
 					writeLines(toString(msg), fileConn)
 					close(fileConn)
-					rm(list=ls())
-					skip <<- TRUE
+					if(file_test("-f",paste0(file,".temporary"))) { file.remove(paste0(file,".temporary")) }
 				})
-#        rm(list=ls())
-#		if(skip) { system(paste("rm ",file,".temporary",sep="")) }
-# RJHD - NH addition for pdf error 2-aug-17                  
-        graphics.off()
+		if(skip) { return(NA) }
+			
+	  # RJHD - NH addition for pdf error 2-aug-17                  
+	  graphics.off()
 		print(paste(file," done",sep=""))
-        rm(list=ls())
+    rm(list=ls())
```

Comment:
By having this here won't all values stored by R be removed? Even though this is the end of the loop there are still values being stored by climpact for use (such as the 'metadata' variable).